api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -45,6 +45,11 @@ message UpstreamLocalityStats {
   // aggregated over all endpoints in the locality.
   uint64 total_error_requests = 4;
 
+  // The total number of requests that were issued by this Envoy since
```

Comment:
How does this relate to `total_requests`? Would be good to integrate with the above comments.

Reply:
the total_requests definition (sum of other total_*_requests) is incorrect above, as in_progress is a gauge.
Suggest removing the comment altogether.

Nit: for consistency it should be total_in_progress_requests .. but i guess we cant do anything about it.

Reply:
Yeah, I agree, the comment is no longer useful now that we have this field and has technical issues. +1 to remove.

Reply:
Done. Please take a look again!